lib/src/main/java/io/github/taz03/jia/responses/friendships/FriendshipsActionResponse.java

```diff
@@ -0,0 +1,8 @@
+package io.github.taz03.jia.responses.friendships;
+
+import io.github.taz03.jia.responses.InstagramResponse;
+import io.github.taz03.jia.responses.models.friendship.Status;
+
+public class FriendshipsActionResponse extends InstagramResponse {
+    private Status friendship_status;
```

Comment:
use java naming convention

Reply:
renamed and setted @JsonProperty annotation